src/components/shared/FilterNavigatorBar.js

```diff
@@ -48,7 +48,9 @@ class FilterNavigatorBar extends PureComponent<Props> {
               title={item}
               onClick={this._onLiClick}
             >
-              <span className="filterNavigatorBarItemContent">{item}</span>
+              <button type="button" className="filterNavigatorBarItemContent">
+                {item}
+              </button>
```

Comment:
Do you think you can do the change of outputting either a span or a button depending on the condition `i === items.length - 1` ?

The idea is that the last element shouldn't likely be a button.